src/styles/blocks/product_card.css

```diff
@@ -0,0 +1,92 @@
+.product-card {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.product-card__product-image {
+  display: block;
```

Comment:
if the parent has display: flex; then all its direct childrens' display prop. will be set to block automatically. no need for this